website/react-magma-docs/src/pages/api/stepper.mdx

```diff
@@ -801,6 +801,97 @@ export function Example() {
 }
 ```
 
+## Vertical
```

Comment:
In addition to this example, lets also show a Vertical stepper inside a dropdown:
![image](https://github.com/user-attachments/assets/6524a91a-fd6c-4ef3-919e-1ab7712fa4ee)